packages/components/src/ConditionalFilter/ConditionalFilter.tsx

```diff
@@ -37,7 +37,7 @@ export interface TextInputProps {
  */
 export interface ConditionalFilterItem {
   id?: string;
-  label?: Node;
+  label?: Element;
```

Comment:
I am curious why Element and not Node. Further more, the item label is used in a `String(capitalize(label))` method. This points toward it being a string type if anything.

Reply:
@Hyperkid123 replaced with `ReactNode`, thank you :slightly_smiling_face: